pkg/minikube/cluster/cluster.go

```diff
@@ -273,20 +245,16 @@ func localkubeURIWasSpecified(config KubernetesConfig) bool {
 }
 
 // SetupCerts gets the generated credentials required to talk to the APIServer.
-func SetupCerts(d drivers.Driver, apiServerName string, clusterDnsDomain string) error {
+func SetupCerts(cmd bootstrapper.CommandRunner, k8s KubernetesConfig) error {
 	localPath := constants.GetMinipath()
-	ipStr, err := d.GetIP()
-	if err != nil {
-		return errors.Wrap(err, "Error getting ip from driver")
-	}
-	glog.Infoln("Setting up certificates for IP: %s", ipStr)
+	ip := net.ParseIP(k8s.NodeIP)
```

Comment:
A little duplication here, 
We already call host.GetIP() in start.go https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/start.go#L143-L147

and populate the KubernetesConfig with that value
https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/start.go#L179